an1.cc

```diff
@@ -684,15 +679,12 @@ static void Write_AN1_Bitmaps(gbfile* f)
 
 static void Read_AN1_Waypoints(gbfile* f)
 {
-  unsigned long count = 0;
-  unsigned long i = 0;
-  an1_waypoint_record* rec = nullptr;
   Waypoint* wpt_tmp;
   char* icon = nullptr;
   ReadShort(f);
-  count = ReadLong(f);
-  for (i = 0; i < count; i++) {
-    rec = Alloc_AN1_Waypoint();
+  unsigned long count = ReadLong(f);
+  for (unsigned long i = 0; i < count; i++) {
+    an1_waypoint_record* rec = Alloc_AN1_Waypoint();
```

Comment:
old line 687-689 declare the variable and initialize it.  The initial value is never used.
old lines 693-695 rewrite all three variables.  The initial values were never read.
new lines 685-687 declare the variables, and use the rewritten values from 693-695, and duplicate the loop.
It looks good to  me.

Reply:
Me, too.  I thought I saw a call to ReadLong(f) being dropped, but on a Real Computer, I see it's not. I suspect the diff just rendered badly on the "small" screen .I read it several times. Sorry for the distraction.